### ISSUE TITLE
../../third_party/dart/runtime/vm/message_snapshot.cc: 576: error: expected: !lib.IsNull()

### ISSUE THREAD

**Issue by jackleemeta** [flutter/flutter#111804]
When performing [SendPort.send], an exception occurred like the issue title.

Flutter SDK  Version: since 3.3.0, but 3.0.5 is ok
flutter mode: `flutter run --release`,  but `flutter run --debug` is ok

**Comment by danagbemava-nc** [comment]
Hi @jackleemeta, please provide a complete minimal reproducible code sample that reproduces the issue along with the full crash log. 

Also, what platform are you experiencing this issue on?

Thank you

**Comment by jackleemeta** [comment]
Hello @danagbemava-nc, the platform is iOS, but I did't save the crash log. And now that I have cleared the build artifact cache for the project, the exception has disappeared.

 I suspect the exception reason is linking to some invalid cache build by older Flutter SDK after Flutter SDK 3.3.2 updated. Anyway, my problem is solved. 

If it is appeared again later, I will provide the reproducible sample and crash log here.

Thanks.
